### PR TITLE
Add Travis CI to release automation docs

### PR DESCRIPTION
**Pull request by jonesphillip**
Add Sentry Travis CI guide to release automation docs.

travis-ci-screenshot

### CODE REVIEW

File: src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          sentry-cli releases finalize $SENTRY_RELEASE
+          sentry-cli releases deploys $SENTRY_RELEASE new -e $SENTRY_ENVIRONMENT
+```
+
+For more details about the release management concepts in the snippet above, see the full documenation on [release management]({%- link _documentation/cli/releases.md -%}).
+
+**Notes**:
+
+- If you’re not deploying a JavaScript project or have sent source maps to Sentry using another method, omit the `upload-sourcemaps` line.
+- If you can’t install a repository integration, send commit metadata via the [create release endpoint]({%- link _documentation/workflow/releases/index.md -%}#alternatively-without-a-repository-integration) or omit the `set-commits` line.
```
Comment (manuzope) [inline src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md:66]: I feel like we should tell folks that if they omit set-commits, they won't get suspect commits. I know we allude to that up top but I feel like also putting it here makes it clear on what they'll be missing out. Thoughts?

File: src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: Travis CI
+sidebar_order: 4
+
+---
+
+This guide walks you through the process of automating Sentry release management and deploy notifications in Travis CI. After deploying in Travis CI, you’ll be able to identify suspect commits that are likely the culprit for new errors. You’ll also be able to apply source maps to see the original code in Sentry.
+
+Before starting, confirm that your Sentry project is properly set up to track commit metadata. The easiest method to do so is to [install a repository integration]({%- link _documentation/workflow/releases/index.md -%}#install-repo-integration). Once that's installed, and you've added your repository, come back to this guide. If you've already installed a repository integration, you're ready to go.
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md:9]:
```suggestion
Before starting, confirm that your Sentry project is properly set up to track commit metadata by [installing a repository integration]({%- link _documentation/workflow/releases/index.md -%}#install-repo-integration). Once that's installed, and you've added your repository, come back to this guide. If you've already installed a repository integration, you're ready to go.
```

File: src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Setting Environment Variables in Travis CI
+
+Next, you'll need a few environment variables to configure the Sentry CLI:
+
+- `SENTRY_AUTH_TOKEN` - Your internal integration token
+- `SENTRY_ORG` - Your Sentry organization slug
+- `SENTRY_PROJECT` - Your Sentry project slug
+
+To access you internal integration token securely, store it as an [environment variable on your repository](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings):
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md:31]:
```suggestion
To access your internal integration token securely, store it as an [environment variable on your repository](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings):
```

File: src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+2. Find the "Environment Variables" section.
+3. Add a new variable with `SENTRY_AUTH_TOKEN` as the name and your internal integration token as the value.
+
+[{% asset releases/travis-ci/travis-env-vars.png alt="View of Travis CI environment variables." %}]({% asset releases/travis-ci/travis-env-vars.png @path %})
+
+The other variables do not contain sensitive information, so you'll use the `env` key to define them as environment variables in your .travis.yml file.
+
+## Create Release and Notify Sentry of Deployment
+
+To automate your Sentry release management process you'll need to add the `Create release and notify Sentry of deploy` job below after your deploy step:
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md:43]:
```suggestion
To automate your Sentry release management process, you'll need to add the `Create release and notify Sentry of deploy` job below after your deploy step:
```

File: src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+          curl -sL https://sentry.io/get-cli/ | bash
+          export SENTRY_RELEASE=$(sentry-cli releases propose-version)
+          sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
+          sentry-cli releases set-commits $SENTRY_RELEASE --auto
+          sentry-cli releases files $SENTRY_RELEASE upload-sourcemaps path-to-sourcemaps-if-applicable
+          sentry-cli releases finalize $SENTRY_RELEASE
+          sentry-cli releases deploys $SENTRY_RELEASE new -e $SENTRY_ENVIRONMENT
+```
+
+For more details about the release management concepts in the snippet above, see the full documenation on [release management]({%- link _documentation/cli/releases.md -%}).
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md:61]:
```suggestion
For more details about the release management concepts in the snippet above, see the full documentation on [release management]({%- link _documentation/cli/releases.md -%}).
```

**Comment by jonesphillip** [comment]
@manuzope @MimiDumpling addressed comments.

**Pull request merged**